delegator/prisma/schema.prisma

```diff
@@ -8,18 +8,59 @@ datasource db {
 }
 
 model Transaction {
-  id          BigInt  @id @default(autoincrement())
-  from        String
-  to          String
-  input       String
-  gas         String
-  value       String
-  chainId     String
-  gasPrice    String
-  nonce       String
-  v           String
-  r           String
-  s           String
-  rawTx       String
-  signedRawTx String?
+  id          BigInt  @id @default(autoincrement()) @map("transaction_id")
+  from        String  @db.VarChar(42)
+  to          String  @db.VarChar(42)
+  input       String  @db.VarChar(1024)
+  gas         String  @db.VarChar(20)
+  value       String  @db.VarChar(20)
+  chainId     String  @db.VarChar(20)
+  gasPrice    String  @db.VarChar(20)
+  nonce       String  @db.VarChar(20)
+  v           String  @db.VarChar(66)
+  r           String  @db.VarChar(66)
+  s           String  @db.VarChar(66)
+  rawTx       String  @db.VarChar(1024)
+  signedRawTx String? @db.VarChar(1024)
+
+  @@map("transactions")
+}
+
+model Organization {
+  id   BigInt @id @default(autoincrement()) @map("organization_id")
+  name String @unique @db.VarChar(50)
+
+  Reporter Reporter[]
+
+  @@map("organizations")
+}
+
+model Contract {
+  id       BigInt     @id @default(autoincrement()) @map("contract_id")
+  address  String     @unique @db.VarChar(42)
+  Function Function[]
+  Reporter Reporter[]
+
+  @@map("contracts")
+}
+
+model Function {
+  id          BigInt   @id @default(autoincrement())
+  name        String   @db.VarChar(100)
+  encodedName String   @db.VarChar(11)
```

Comment:
Can you please explain why did you decide to set the length to 11? I think the 10 should be the right length.